src/components/AdoptionProcess/ProcessContainer.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react'
+import ProcessItems from "./ProcessItems"
+import { Row, Col } from 'react-bootstrap';
+
+const ProcessContainer = () => {
+    const adoptionProcess = [
```

Comment:
this is wrong, the array should come from props and then using the props u will call the map to render the items

Reply:
Hi halit, I made the array of object as props and pushed the changes. can you please recheck ?